java/vector/src/test/java/org/apache/arrow/vector/TestViewVarBinaryVector.java

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
I would recommend to keep them in the `TestVarCharViewVector` for the moment. I know the naming is not so matching here. For other `VarBinaryView` tests we used the same test class. And I see it is not very matching. Maybe we could rename the class to `TestVariableWidthViewVector`? This is an issue from my part in the initial PR. We can fix it here.

Reply:
Rename the test class to `TestVariableWidthViewVector` and move `testSetNullableViewVarBinaryHolder` to it.